tests/pipeline/nodes/model/movenetv1/test_predictor.py

```diff
@@ -162,38 +143,10 @@ def test_get_results_single(self, movenet_config, model_dir):
             keypoints_scores_no_pose,
             keypoints_conns_no_pose,
         ) = movenet_predictor._get_results_single(prediction_no_pose)
-        npt.assert_array_equal(
-            x=bbox_no_pose,
-            y=np.zeros(0),
-            err_msg=(
-                "Unexpected bbox output for prediction with keypoint score below "
-                f"threshold got {bbox_no_pose} instead of {np.zeros(0)}"
-            ),
-        )
-        npt.assert_array_equal(
-            x=valid_keypoints_no_pose,
-            y=np.zeros(0),
-            err_msg=(
-                "Unexpected valid keypoint output for prediction with keypoint score "
-                f"below threshold got {valid_keypoints_no_pose} instead of {np.zeros(0)}"
-            ),
-        )
-        npt.assert_array_equal(
-            x=keypoints_scores_no_pose,
-            y=np.zeros(0),
-            err_msg=(
-                "Unexpected keypoint score output for prediction with keypoint score "
-                f"below threshold got {keypoints_scores_no_pose} instead of {np.zeros(0)}"
-            ),
-        )
-        npt.assert_array_equal(
-            x=keypoints_conns_no_pose,
-            y=np.zeros(0),
-            err_msg=(
-                "Unexpected keypoint connection output for prediction with keypoint "
-                f"score below threshold got {keypoints_conns_no_pose} instead of {np.zeros(0)}"
-            ),
-        )
+        npt.assert_array_equal(bbox_no_pose, np.empty((0, 4)))
+        npt.assert_array_equal(valid_keypoints_no_pose, np.empty(0))
+        npt.assert_array_equal(keypoints_scores_no_pose, np.empty(0))
+        npt.assert_array_equal(keypoints_conns_no_pose, np.empty(0))
```

Comment:
Similar to above comment that updated codes seem to dilute the error messages.
In this case, the original error messages also indicate where/what the error is `keypoint connection` vs `keypoint score` vs etc...

Reply:
pytest error logs include the code for the test function which fails and additional `err_msg` does not seem to provide more information than what the array contents, comments about expected output and variable naming already provide.

Error log with original `err_msg`
```
        npt.assert_array_equal(
            x=bbox_no_pose,
            y=np.zeros(0),
            err_msg=(
>               "Unexpected bbox output for prediction with keypoint score below "
                f"threshold got {bbox_no_pose} instead of {np.zeros(0)}"
            ),
        )
E       AssertionError: 
E       Arrays are not equal
E       Unexpected bbox output for prediction with keypoint score below threshold got [1.] instead of []
E       (shapes (1,), (0,) mismatch)
E        x: array([1.])
E        y: array([], dtype=float64)

tests/pipeline/nodes/model/movenetv1/test_predictor.py:151: AssertionError
```

Error logs without `err_msg`
```
>       npt.assert_array_equal(bbox_no_pose, np.empty((0, 4)))
E       AssertionError: 
E       Arrays are not equal
E       
E       (shapes (1,), (0, 4) mismatch)
E        x: array([1.])
E        y: array([], shape=(0, 4), dtype=float64)

tests/pipeline/nodes/model/movenetv1/test_predictor.py:155: AssertionError
```
As the pytest error logs include the `assert` function, it can be seen that the test failed because `bbox_no_pose` is not equal to `np.empty((0, 4))`.